### DISCUSSION TITLE
Domain blocked as identified wongly as phishing site

### DISCUSSION THREAD

**Discussion by christofkac** [dani-garcia/vaultwarden#2353] [Q&A]
I was yesterday informed by my dns provider, that my vaultwarden site was identified by as a phishing site and therefore blocked.

Here the mail which I received mentioning that we use the bitwarden logo and therefore breach a trademark:

> Hello,
> 
> Please be informed that the chrkac [.] com domain name was reported as involved in abusive activity by a trusted organization. During the investigation, it was noticed that your website content is a copy of the Bitwarden official website. On that ground, we were forced to suspend the domain name due to phishing activities, which include unauthorized use of the legitimate organization denomination and attempts to acquire sensitive information such as usernames, passwords, etc.
> 
> As the domain name contains a trademark and uses a logo of the registered brand, for us to be sure that it was registered for legitimate purposes only, you will need to provide us with paperwork proving your cooperation with the Bitwarden website and their consent to use their official denomination in your domain name. Such evidence may be an official document that outlines your relation to official business.
> 
> Alternatively, an official representative of Bitwarden can email us directly via legalandabuse@namecheap.com with a claim that you have the lawful ground to act on their behalf.
> 
> The paperwork should be scanned, converted to PDF format, and sent to us via email.
> 
> Thank you for understanding. Looking forward to hearing from you

I've seen on reddit.com that I'm not alone: [https://www.reddit.com/r/Bitwarden/comments/t4n84j/my_bitwarden_instance_has_been_flagged_as_a/](url)

Any idea how to proceed?

**Comment by BlackDex** [comment]
For a company i work for we receive these kind of message almost every week. That company sells Spotify, Google Play Cards, Apple iTunes cards etc.. etc.. And we receive almost every week (sometimes less) a mail about using a logo which is trademarked while those sites are trying to sell products which provide money for them.

Also, some rouge companies even pretend to act in the name of the trademark holder, while try only try to scam you into paying some kind of fee or fine or whatever.

Regarding the phishing, i think that is more a browser issue, or maybe a anti-virus software or maybe even some firewalls.
I'm not sure what you think this project or for that matter even Bitwarden can do regarding matching on a domain name and flagging it as potential harmful.

If using a different sub-domain works, then i suggest to do that, and report this false positive to the tool which is marking it as unsafe.

**Reply by mbosch** [reply]
I may have to switch mine to a sub-domain on a different domain and hope it gets less attention.  Currently I have a .com that I dedicate to hosting my VaultWarden server. I went from 0 scanners reporting my domain as phishing/malicious on virustotal to 13 in a matter of days.  I submitted false positive cases with all of them last night but there are 90 or so scanners that virustotal aggregates so I'm hoping the others don't just pile on.  

I know it's not VW's fault or anything but hoping that others might be able to share what worked for them.

**Comment by jjlin** [comment]
The "trusted organization" referenced in the email is presumably [Google Safe Browsing](https://safebrowsing.google.com/), and you can see that your domain is currently listed as unsafe: https://transparencyreport.google.com/safe-browsing/search?url=chrkac.com

You could probably start by reporting your site as a false positive at https://safebrowsing.google.com/safebrowsing/report_error/, explaining that Bitwarden provides their software for users to host on their own servers (https://bitwarden.com/help/install-on-premise-linux/#installation-procedure), and they even suggest using a subdomain of the form "bitwarden.example.com". If you can get your domain off their list, that may help convince Namecheap that it's a mistake. You could also just provide the same explanation directly to Namecheap, though whoever reads it might not even understand the explanation.

Ultimately, this is Google's fault for not accounting for the fact that Bitwarden can be self-hosted, and Namecheap's fault for blindly accepting whatever Safe Browsing claims. Perhaps you should look into a registrar that is less likely to do that.

**Reply by christofkac** [reply]
Thanks for the hint. I contacted google - let's see how quickly they respond.

**Reply by paradix** [reply]
I have the same issue with google and some other services marking my domain as phishing. 
Did you manage to unflag yours?

**Comment by popindavibe** [comment]
Looks like something changed suddenly at Google Safe Browsing, my domain got flagged yesterday as well. I haven't received any notification emails on my abuse mailbox, and so far the few request to unlock that I've posted haven't changed anything :disappointed:

**Comment by pepa65** [comment]
Ours got flagged yesterrday, and I just reported it as a Safe domain, and less than 24h later it was back to normal.

**Reply by paradix** [reply]
Where did you report it. My reports seem to get ignored :(

**Reply by christofkac** [reply]
I reported it on the above mentioned site https://safebrowsing.google.com/safebrowsing/report_error/ and my site was also unblocked within less than 24h

**Comment by pepa65** [comment]
I think I clicked on More Information or something like that, and then there was a button This is not Unsafe, which I clicked, which landed me on a Google form where I could put some words in why it is not unsafe.

**Comment by popindavibe** [comment]
I reported it to that same URL multiple times, and after 3 weeks google did... f**k all.
-- Sent from K9-Mail

On 1 April 2022 12:52:27 CEST, christofkac ***@***.***> wrote:
>I reported it on the above mentioned site
>https://safebrowsing.google.com/safebrowsing/report_error/ and my site
>was also unblocked within less than 24h
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/dani-garcia/vaultwarden/discussions/2353#discussioncomment-2485009
>You are receiving this because you commented.
>
>Message ID:
***@***.***>

**Reply by paradix** [reply]
Same here, absolutely no reaction. Mine still gets blocked and not only by Google, also MS Defender Smart Screen marks as unsecure.

**Reply by mbosch** [reply]
Same. I was able to get a few sites to unblock me but they just kept multiplying.  I gave up and just imported my logins into the hosted BW.  It's too much work to try and play whack a mole with them.

**Reply by BlackDex** [reply]
Or, as others suggest, change the domain name, just do not use bitwarden in the domain, use vaultwarden, or vault, or vw, or something else.

**Reply by paradix** [reply]
> Or, as others suggest, change the domain name, just do not use bitwarden in the domain, use vaultwarden, or vault, or vw, or something else.

In general yeah that's what I intend to do, but the fu**ers flagged my whole domain

**Reply by popindavibe** [reply]
A quick update on this after 6 months:
- Google finally delisted us after 2-3 months.
- Among the 10 antivirus / protection solution that blindly flagged us when Google did, [5 are still blocking us](https://atrium.forum.nomagic.uk/d/5-google-safe-browsing-is-blacklisting-nomagic-domains/10).

**Comment by HammyHavoc** [comment]
Experiencing this issue too with the splitanatom.com domain as a whole. Can't save changes I make in the Bitwarden browser extension. Extremely frustrating.

What's the workaround in the meantime?

**Comment by popindavibe** [comment]
The workaround is to create an alias (A / AAAA or just plain CNAME DNS entry) for your website.
You set up your web server to accept that alias for your Vaultwarden vhost, et voilà!

I don't know how long it will work (you probably **have to keep contacting Google Safe Browsing for delisting**, as it's followed by many security products and browsers) but for us, this allowed people stuck with corporate-locked computer still blacklisting us after Google delisting to access our Vaultwarden.
-- Sent from K9-Mail

On 9 September 2022 02:32:23 CEST, Hammy Havoc ***@***.***> wrote:
>Experiencing this issue too with the splitanatom.com domain as a whole.
>Can't save changes I make in the Bitwarden browser extension. Extremely
>frustrating.
>
>What's the workaround in the meantime?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/dani-garcia/vaultwarden/discussions/2353#discussioncomment-3602505
>You are receiving this because you commented.
>
>Message ID:
***@***.***>
